### ISSUE TITLE
some previews / thumbnails aren't rendering

### ISSUE THREAD

**Issue by Meteor0id** [SimpleMobileTools/Simple-Gallery#3062]
Versie: 6.28.1
Besturingssysteem: 10
Hi, I noticed some previews aren't rendering for all kinds of media (videos, gifs, movie). Clearing cache made no difference. These media can still be viewed properly by tapping and opening single view mode.
I have attached one of the images for which the app fails to show a preview in folder view.

Hope this helps.

Attachment was too large here is a link
https://file.io/qY4JBLr03tWf

Also can't email hello@simplemobiletools.com the mail doesnt get delivered.

**Comment by inson1** [comment]
SMT apps were sold to ad company. Fork is here https://github.com/FossifyOrg

**Comment by Meteor0id** [comment]
Yup I know, and so we all had trackers on our phones because one guy sold us out and did not pop up a notice for changed terms and conditions. That's criminal, lets complaint with the relevant autority.
